source/documentation/publish_and_manage_data/add_data.md

```diff
@@ -35,6 +35,8 @@ Some organisations, such as intelligence agencies, are exempt from publishing or
 
 Local authorities must publish senior staff data in a [different format](http://schemas.opendata.esd.org.uk/OrganisationStructure).
 
+For detailed policy guidance about publishing organograms, email the Cabinet Office transparency team at transparencydatateam@cabinetoffice.gov.uk.
```

Comment:
In line with style guide, I'd recommend making the email an active link

Reply:
Done 👌

Reply:
👍